hal/tpm_io.c

```diff
@@ -144,6 +144,8 @@ int TPM2_IoCb(TPM2_CTX* ctx, INT32 isRead, UINT32 addr,
         ret = TPM2_IoCb_STCubeMX_I2C(ctx, isRead, addr, buf, size, userCtx);
     #elif defined(CY_USING_HAL)
         ret = TPM2_IoCb_Infineon_I2C(ctx, isRead, addr, buf, size, userCtx);
+    #elif defined(WOLFSSL_ESPIDF)
```

Comment:
Above in includes at line 72 please add:

```
#elif defined(WOLFSSL_ESPIDF)
#include "hal/tpm_io_espressif.c"
```

Reply:
ah, yes. Good catch. Not needed for this cmake, but appropriate for consistency and possibly alternative environments.